client/js/navigation/NavigationLayout.ios.js

```diff
@@ -0,0 +1,99 @@
+import React from "react";
+import { Platform } from "react-native";
+import {
+    createStackNavigator,
+    createBottomTabNavigator
+} from "react-navigation";
+import MainPageScreen from '../screens/MainPage';
+import PosesScreen from '../screens/Poses';
+import ProfileScreen from '../screens/Profile';
+import PosesDetailsScreen from '../screens/PosesDetails';
+import WorkoutSessionScreen from '../screens/WorkOutSession';
+import DailyReportsScreen from '../screens/DailyReports';
+import CompletedChallenegesScreen from '../screens/CompletedChallenges';
+import RemindersScreen from '../screens/Reminders';
+import AboutScreen from '../screens/About';
+import SignInScreen from '../screens/SignIn';
+import EditProfileScreen from '../screens/EditProfile';
+import SignUpScreen from '../screens/SignUp'
+import { sharedNavigationOptions } from "./config";
+import Ionicons from "react-native-vector-icons/Ionicons";
+
+const MainStack = createStackNavigator(
+    {
+    Main: MainPageScreen,
+    WorkoutSession: WorkoutSessionScreen
+        
+    },
+    {
+        defaultNavigationOptions: ({ navigation }) => ({
+            ...sharedNavigationOptions(navigation)
+        })
+    }
+);
+
+
+const PosesStack = createStackNavigator(
+  {
+    Poses: PosesScreen,
+    PosesDetails: PosesDetailsScreen
+  },
+  {
+    defaultNavigationOptions: ({ navigation }) => ({
+      ...sharedNavigationOptions(navigation)
+    })
+  }
+);
+
+const ProfileStack = createStackNavigator(
+  {
+    Profile: ProfileScreen,
+    DailyReports: DailyReportsScreen,
+    CompletedChalleneges: CompletedChallenegesScreen,
+    Reminders: RemindersScreen,
+    About: AboutScreen,
+    EditProfile: EditProfileScreen,
+    SignIn: SignInScreen,
+    signUp: SignUpScreen
```

Comment:
I believe that SignIn and SignUp should be in the auth stack. Not here. Please make the changes.